tests/app/template/test_rest.py

```diff
@@ -616,6 +616,37 @@ def test_update_set_process_type_on_template(client, sample_template):
     assert template.process_type == 'priority'
 
 
+def test_create_a_template_with_reply_to(client, sample_user):
+    service = create_service(service_permissions=['letter'])
+    letter_contact = create_letter_contact(service, "Edinburgh, ED1 1AA")
+    data = {
+        'name': 'my template',
+        'subject': 'subject',
+        'template_type': 'letter',
+        'content': 'template <b>content</b>',
+        'service': str(service.id),
+        'created_by': str(sample_user.id),
+        'reply_to': str(letter_contact.id),
+    }
+    data = json.dumps(data)
+    auth_header = create_authorization_header()
+
+    response = client.post(
+        '/service/{}/template'.format(service.id),
+        headers=[('Content-Type', 'application/json'), auth_header],
+        data=data
+    )
+    assert response.status_code == 201, response.get_data(as_text=True)
+
+    json_resp = json.loads(response.get_data(as_text=True))
+    assert json_resp['data']['template_type'] == 'letter'
+    assert json_resp['data']['reply_to'] == str(letter_contact.id)
+
+    template = Template.query.get(json_resp['data']['id'])
+    from app.schemas import template_schema
```

Comment:
can this go up the top with the rest of the imports?

Reply:
I originally copied it from the test above and it looks like all tests delay this import like this.
It turns out 59 tests fail if this is moved to module level. My guess is marshmallow stores some db/session reference somewhere and if that happens before the session fixtures are applied (eg during test module import) it gets stuck with a `DummySession`.

I'm not really sure what the implications of this are.

Reply:
😩